src/DataTable/DataTable.jsx

```diff
@@ -14,38 +14,46 @@ export class DataTable extends Component {
   }
   
   static defaultProps = {
-    autoResize: false
+    autoResize: true
   }
-  
-  handleResize (e) {
+
+  handleResize (grid) {
     if (this.autoResize) {
-      this.refs.grid.api.sizeColumnsToFit()
+      grid.api.sizeColumnsToFit()
     }
   }
-
+  
   componentDidMount () {
-    window.addEventListener('resize', this.handleResize.bind(this))
+    const grid = this.refs.grid
+  
+    if (this.autoResize) {
+      grid.api.sizeColumnsToFit()
+    }
+  
+    this.boundResize = this.handleResize.bind(this, grid)
+  
+    window.addEventListener('resize', this.boundResize)
   }
-
+  
   componentWillUnmount () {
-    window.removeEventListener('resize', this.handleResize.bind(this))
+    window.removeEventListener('resize', this.boundResize)
   }
-
+  
   headerCellRendererFunc (params) {
     const api = params.api
     const cb = document.createElement('input')
     cb.setAttribute('type', 'checkbox')
     cb.setAttribute('class', 'ag-selection-checkbox')
-
+  
     const label = document.createElement('label')
     const title = document.createTextNode(params.colDef.headerName)
     const span = document.createElement('span')
     span.setAttribute('class', 'ag-cell-value')
-    
+  
     label.appendChild(cb)
     label.appendChild(span)
     label.appendChild(title)
-    
+  
     cb.addEventListener('change', function (e) {
       if (this.checked) {
         api.selectAll()
```

Comment:
The final return in the render function has the jsx indented too much. its 4 spaces instead of 2.